<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogLogin</class>
 <widget class="QDialog" name="QDialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>389</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="13,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="backGround">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/images/images/background.bmp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="userName">
       <property name="text">
        <string>用 户 名</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item>
      <widget class="QLabel" name="passWord">
       <property name="text">
        <string>密  码</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPSWD"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkSaveName">
       <property name="text">
        <string>保存用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>确  定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>取  消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>QDialogLogin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
